### PR TITLE
[MERGE] digest, various: improve digest email layout and content

### PR DESCRIPTION
**Pull request by parthchoksi-github**
Purpose of this merge is to provide improvements in digest emails layout

- Added "Connect" button to connect to the database.
- Changed the messages count which will now only consider real messages and not notes, notifications etc.
- Fixed some margin , padding issues and minor message improvements.
- Added a tip for administration users which will be sequentially sent first to them.
- Moved recipients to a new tab.
- Changed to company's email as email author.
- Changed '>' to greaterthan and '<' to lessthan in default template. (according to updated Jinja2)
- In one of the tip, we are showing all kinds of users in the list. So, it was replaced with only internal active users.

Related to task 1918364

### CODE REVIEW

File: addons/digest/models/digest.py [hunk 1]
```diff
@@ -65,9 +65,10 @@ def _compute_kpi_res_users_connected_value(self):
             record.kpi_res_users_connected_value = user_connected
 
     def _compute_kpi_mail_message_total_value(self):
+        discussion_subtype = self.env.ref('mail.mt_comment')
```
Comment (dbeguin) [inline addons/digest/models/digest.py]: As this is only used once to get its id, you could directly get the id.
`discussion_subtype_id = self.env.ref('mail.mt_comment').id`

File: addons/digest/models/digest.py [hunk 2]
```diff
@@ -183,17 +184,17 @@ def _get_margin_value(self, value, previous_value=0.0):
     def _format_currency_amount(self, amount, currency_id):
         pre = post = u''
         if currency_id.position == 'before':
-            pre = u'{symbol}\N{NO-BREAK SPACE}'.format(symbol=currency_id.symbol or '')
+            pre = u'{symbol}'.format(symbol=currency_id.symbol or '')
         else:
-            post = u'\N{NO-BREAK SPACE}{symbol}'.format(symbol=currency_id.symbol or '')
+            post = u'{symbol}'.format(symbol=currency_id.symbol or '')
```
Comment (dbeguin) [inline addons/digest/models/digest.py]: I would also simplify this : 
```
pre = currency_id.position == 'before'
symbol = u'{symbol}'.format(symbol=currency_id.symbol or '')
return u'{pre}{0}{post}'.format(amount, pre=symbol if pre else '', post=symbol if not pre else '')
```

File: addons/digest/models/digest.py (hunk 1 above)
Comment (pch-odoo) [reply addons/digest/models/digest.py]: Done. :+1:

File: addons/digest/models/digest.py (hunk 2 above)
Comment (pch-odoo) [reply addons/digest/models/digest.py]: Done. Thanks :+1:

File: addons/digest/data/digest_data.xml [hunk 3]
```diff
@@ -11,6 +11,19 @@
     </data>
 
     <data>
+
+        <record id="digest_tip_mail_2" model="digest.tip">
+            <field name="group_id" ref='base.group_system'/>
+            <field name="sequence">1</field>
```
[the remark this answers is not in the document]
Comment (pch-odoo) [reply addons/digest/data/digest_data.xml]: Okay, Changed the sequences. :+1:

File: addons/digest/data/digest_template_data.xml [hunk 4]
```diff
@@ -3,8 +3,8 @@
     <record id="digest_mail_template" model="mail.template">
         <field name="name">Digest: Default main template</field>
         <field name="model_id" ref="digest.model_digest_digest"/>
-        <field name="email_from">${user.email_formatted | safe}</field>
         <field name="subject">${'%s: %s' % (ctx.get('user', user).company_id.name, object.name)}</field>
+        <field name="email_from">${('"%s" &lt;%s&gt;' % (object.company_id.name, object.company_id.email or user.email_formatted))|safe}</field>
```
Comment (tde-banana-odoo) [inline addons/digest/data/digest_template_data.xml]: I think you could use object.company_id.partner_id.email_formatted or user.email_formatted. Could you give a try ?

File: addons/digest/data/digest_data.xml [hunk 5]
```diff
@@ -41,5 +41,18 @@
...
         </record>
+
+        <record id="digest_tip_mail_2" model="digest.tip">
+            <field name="group_id" ref='base.group_system'/>
+            <field name="sequence">0</field>
+            <field name="tip_description" type="html">
+<div>
+    <div style="font-size: 14px;">
+        To reach your full potential with Odoo, have a look at our <a href="https://www.odoo.com/page/tour">apps videos</a> and discover new tools to work better and faster.
+    </div><br/>
```
Comment (tde-banana-odoo) [inline addons/digest/data/digest_data.xml]: Why 2 nested div and an unnecessary br ?

Comment (pch-odoo) [reply addons/digest/data/digest_data.xml]: Alright, modified it. :+1:

File: addons/digest/data/digest_template_data.xml (hunk 4 above)
Comment (pch-odoo) [reply addons/digest/data/digest_template_data.xml]: Yep. Works! Done :+1:

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-merge

**Comment by tde-banana-odoo** [comment]
@robodoo retry

**Pull request closed without merging**